.github/workflows/py-ci.yml

```diff
@@ -163,7 +163,7 @@ jobs:
           - status_reports
           - tabular_preview
           - transcode
-        python-version: ['3.7']
+        python-version: ['3.9']
```

Comment:
What's the purpose of changes in this file?

Reply:
This was me messing around trying to figure out why the build was broken. I will start reverting them...

Reply:
reverted